Lib/json/tool.py

```diff
@@ -27,23 +46,28 @@ def main():
                         help='write the output of infile to outfile')
     parser.add_argument('--sort-keys', action='store_true', default=False,
                         help='sort the output of dictionaries alphabetically by key')
+    parser.add_argument('--jsonlines', action='store_true', default=False,
+                        help='Parse output using the jsonlines format.')
```

Comment:
I think there are three typos here: `Parse output` -> `parse input` (the other help texts are not capitalized) and the other texts don't end in a period.